### PR TITLE
Add training datastore files and change the bash files accordingly

### PR DESCRIPTION
**Pull request by lucky-suman**
Training Data Pipeline contains:
- Communication with s3.(Tested on Minio Server.)
- Able to fetch Keywords from Github Graph QL.
- Able to extract dependencies for the latest version of the package.
- Extract Vocabulary and first level dependencies successfully.

### CODE REVIEW

**Comment by rootAvish** [comment]
Do you want to write any unit tests for this?

File: training/datastore/utils.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from rudra import logger
+import numpy as np
+import pandas as pd
+import json
+import io
+
+
+class Utility:
```
Comment (rootAvish) [inline training/datastore/utils.py]: Utilities don't need a class of their own IMO, because all methods would be static anyway.

**Comment by lucky-suman** [comment]
> Do you want to write any unit tests for this?

Yes, I started writing the test cases.

**Comment by tisnik** [comment]
Why is the unit test cover coverage so low please?

**Comment by rootAvish** [comment]
@tisnik Codecov is acting out for some reason? IDK, Lucky hasn't touched the files it says have gone down in coverage. Pardon me if I'm just reading the report wrong.

**Comment by tisnik** [comment]
@rootAvish actually the problem is somewhere else, because at least one unit test failed:

```platform linux -- Python 3.6.6, pytest-4.3.0, py-1.7.0, pluggy-0.8.1 -- /usr/bin/python3.6
cachedir: .pytest_cache
rootdir: /tests/unit_tests, inifile:
plugins: cov-2.6.1
collecting ... collected 9 items / 2 errors / 7 selected

==================================== ERRORS ====================================
____________________ ERROR collecting test_get_keywords.py _____________________
ImportError while importing test module '/tests/unit_tests/test_get_keywords.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
usr/local/lib/python3.6/site-packages/_pytest/python.py:500: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
usr/local/lib/python3.6/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
usr/local/lib/python3.6/site-packages/_pytest/assertion/rewrite.py:296: in load_module
    six.exec_(co, mod.__dict__)
tests/unit_tests/test_get_keywords.py:23: in <module>
    import pandas as pd
E   ModuleNotFoundError: No module named 'pandas'
________________________ ERROR collecting test_utils.py ________________________
ImportError while importing test module '/tests/unit_tests/test_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
usr/local/lib/python3.6/site-packages/_pytest/python.py:500: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
usr/local/lib/python3.6/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
usr/local/lib/python3.6/site-packages/_pytest/assertion/rewrite.py:296: in load_module
    six.exec_(co, mod.__dict__)
tests/unit_tests/test_utils.py:24: in <module>
    from training.datastore.utils import Utility
training/datastore/utils.py:23: in <module>
    import pandas as pd
E   ModuleNotFoundError: No module named 'pandas'
```

So - first we need to MAKE SURE the failing tests cause the CI failure. second it needs to be fixed, of course :)

**Comment by tisnik** [comment]
(actually I expect everybody run unit tests before PR :)

**Pull request merged**